Source/WebCore/dom/ViewTransition.cpp

```diff
@@ -83,7 +83,7 @@ void ViewTransition::skipViewTransition(ExceptionOr<JSC::JSValue>&& reason)
     ASSERT(m_document->activeViewTransition() == this);
     ASSERT(m_phase != ViewTransitionPhase::Done);
 
-    if (m_phase == ViewTransitionPhase::PendingCapture) {
+    if (m_phase == ViewTransitionPhase::PendingCapture || m_phase == ViewTransitionPhase::Capturing) {
```

Comment:
```suggestion
    if (m_phase < ViewTransitionPhase::UpdateCallbackCalled) {
```
Can we write it like this to match the spec?

Reply:
Done.